qnm/angular.py

```diff
@@ -118,7 +157,28 @@ def l_min(s, m):
     return np.max([np.abs(s), np.abs(m)])
 
 def M_matrix(s, c, m, l_max):
-    """ TODO Document """
+    """  Symmetric pentadiagonal matrix of size N x N for 
```

Comment:
This can be more descriptive, e.g. "Spherical-spheroidal decomposition matrix, truncated at l_max."